lib/cocoapods-core/podfile/target_definition.rb

```diff
@@ -140,7 +140,7 @@ def name
       # Sets the path of the user project this target definition should link
       # with.
       #
-      # @param  [String] path
+      # @param  [name] name
```

Comment:
why this change?

Reply:
This did not match the parameter variable. Its a warning fix

Reply:
apologies, I meant the `[String]` -> `[name]` portion. shouldn't it still be `[String]`?

Reply:
lol!